ptrace/cpu_info.py

```diff
@@ -39,7 +39,8 @@
     CPU_PPC64 = (_machine in ('ppc64', 'ppc64le'))
     CPU_I386 = (_machine in ("i386", "i686"))    # compatible Intel 32 bits
     CPU_X86_64 = (_machine in ("x86_64", "amd64"))  # compatible Intel 64 bits
-    CPU_ARM = _machine.startswith('arm')
+    CPU_ARM32 = _machine.startswith('arm')
+    CPU_ARM64 = _machine.startswith('aarch64')
```

Comment:
Architecture name is aarch64 so '_machine == 'aarch64' should just work.

32-bit arm can be arm4/v4/v5/v5te/v6/v7/v8 etc so that's why you need .startswith.

Reply:
Thank you for your input, it has been reworked and renamed !